src/interfaces/payment-interfaces/fees-interface.ts

```diff
@@ -16,7 +16,7 @@ export interface TaxesIF {
   gst: number
 }
 
-export const EmptyFees: FeesIF = {
+export const EmptyFees: Array<FeesIF> = [{
```

Comment:
Since FeesIF is a single object, I think EmptyFees should be a single object too. You can move the array declaration to where this object is assigned.

Reply:
oh, yes. You're right. Thanks!